### ISSUE TITLE
Domain

### ISSUE THREAD

**Issue by kdungs** [lhcb/starterkit#7]
We could get something like `lhcb-starterkit.web.cern.ch` with little effort I think.
Ideally `lhcb-starterkit.cern` but I have no idea how to go about that.

**Comment by betatim** [comment]
Sounds interesting. Can it be setup to work with gh-pages?

**Comment by ibab** [comment]
Github pages supports custom domain names through CNAME records.
Not sure if we have that much control over the CERN DNS though.

**Comment by alexpearce** [comment]
It would be great to have `lhcb-starterkit.cern`, or even `starterkit.cern`.

There's more information on registering on the [`home.cern` announcement page](http://home.cern/cern-people/announcements/2015/10/cern-website-moves-homecern). Essentially, you open a ticket along with a “sponsor”, which should be the:

> name of a person who represents a recognised CERN entity. e.g. spokesperson for experiment

We could contact Guy about this.

The other requirements we need to consider are in the [`.cern` registration policy document](http://nic.cern/registration-policy/). They're all straight-forward. I think the only change we would need to make to the site, as is it today, is to add the CERN logo.

**Comment by kdungs** [comment]
👍  for `starterkit.cern`

**Comment by apuignav** [comment]
I think I prefer `lhcb-starterkit.cern`, will be easier to justify 😄

**Comment by kdungs** [comment]
That for sure but I was thinking about The Future™.
But seriously, if we want to create "a brand" and be "thoughtleaders" (so many 💩 buzzwords), Starterkit™ would be a better option imho.

**Comment by apuignav** [comment]
Fair enough. I never stop thinking about The Future:tm: (too much work!). Let's see what others say!

**Comment by kdungs** [comment]
Also for purely strategical reasons: If we can get `starterkit.cern` it should be easy to _also_ get `lhcb-starterkit.cern` or just have it as a fallback. (They might even, if they are not happy with `starterkit` suggest we use `lhcb-starterkit`). So aim for the bigger 🐟  :)

**Comment by apuignav** [comment]
We got authorization/sponsorship from the management (also affects #18). Time to prepare the ticket!
